### PR TITLE
asteroid collision problem sol

### PR DESCRIPTION
**Pull request by Sahib573**

### CODE REVIEW

**Review by nimishbongale — COMMENTED** [review]
Are both these files empty? You might want to have a look!

**Comment by Sahib573** [comment]
please review it again.

File: Task 1/asteroid_collision/solution.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    {
+        ans.push_back(s.top());
+        s.pop();
+    }
+    reverse(ans.begin(), ans.end());
+    for (auto val : ans)
+        cout << val << "  ";
+
+    return 0;
+}
```
Comment (nimishbongale) [inline Task 1/asteroid_collision/solution.cpp:46]: Do add a newline here at EOF

**Review by nimishbongale — CHANGES_REQUESTED** [review]
The question.md file is still empty

File: Task 1/asteroid_collision/solution.cpp (hunk 1 above)
Comment (Sahib573) [reply Task 1/asteroid_collision/solution.cpp]: added new line

**Comment by nimishbongale** [comment]
> The question.md file is still empty

Do look into this

**Review by nimishbongale — APPROVED** [review]
Thanks for contributing! 🚀

**Pull request merged**